core/templates/dev/head/exploration_editor/main_tab/StateResponses.js

```diff
@@ -306,6 +308,26 @@ oppia.controller('StateResponses', [
         return choiceIndices.every(function(choiceIndex) {
           return handledAnswersArray.indexOf(choiceIndex) !== -1;
         });
+      } else if (interactionId === 'ItemSelectionInput') {
```

Comment:
I think you only want these checks to apply if the ItemSelectionInput interaction is in "single-choice" mode, so you'll need another check for the correct customization arg (i.e., max number of answers that can be entered must be 1).